safegraph/delphi_safegraph/run.py

```diff
@@ -24,11 +24,14 @@ def run_module():
     aws_endpoint = params["aws_endpoint"]
     wip_signal = params["wip_signal"]
 
-    process_file = partial(process,
```

Comment:
I noticed this used `partial` before and no longer -- does that make any difference? (I don't understand why you would use that over a regular function)

Reply:
I don't think `partial` is useful here anymore.  It was used to fix all the other arguments to `process()` since because `pool.map()` requires the function to have a single argument.  With the addition of the smoothing, we need two variable arguments to `process()`:  the current filename and the list of previous filenames (derived from the current filename).  It would be possible to refactor the call to `files_in_past_week` to live inside of `process()` but that strikes me as harder to test and not as easily extendable to #273.

Reply:
Thanks for the explanation!